models/user.js

```diff
@@ -2,7 +2,7 @@
 
 const bcrypt = require('bcrypt');
 const mongoose = require('mongoose');
-const validator = require('validator');
+const emailValidator = require('email-validator');
```

Comment:
Ok, this is way better than previous solution, keep it as is but just for sake of argument next time take author background into account. For example, I personally tend to have more faith in:

- hapijs folks - https://github.com/hapijs/isemail
- [segment.io](https://segment.com/) - https://github.com/segmentio/is-email

Reply:
Valid point, I'll keep this in mind.